packages/netlify-cms-widget-relation/src/RelationControl.js

```diff
@@ -82,14 +82,17 @@ export default class RelationControl extends React.Component {
     }
   }
 
-  componentWillReceiveProps(nextProps) {
+  componentDidUpdate(prevProps) {
+    /**
+     * Load extra post data into the store after first query.
+     */
```

Comment:
[Invoke external callbacks in `componentDidUpdate`](https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html#invoking-external-callbacks)